syntax/parse.rs

```diff
@@ -214,6 +224,15 @@ fn parse_enum(cx: &mut Errors, item: ItemEnum, namespace: &Namespace) -> Api {
         cx.error(where_clause, "enum with where-clause is not supported");
     }
 
+    let repr = match repr {
+        Some(Repr::Atom(atom)) => Some(atom),
+        Some(Repr::Align(_)) => {
+            cx.error(&item, "repr(align) on enums is not supported");
```

Comment:
It should be clarified that this is a limitation of C++ enums and not cxx.

```suggestion
            cx.error(&item, "C++ does not support custom alignment on an enum");
```

Reply:
Also the error should point to just the align attribute, not the entire item.